### PR TITLE
Array Extensions Unit Tests

### PR DESCRIPTION
**Pull request by jwicks31**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes
- Unit Tests for Array Extensions

Part of #120

Added Unit Tests for the Extensions on Array.prototype

### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

**Review by lautarodragan — DISMISSED** [review]
* [ ] Tested changes manually ([N/A])
* [X] Checked accidental architectural/style changes
* [X] Reviewed entire diff
* [X] Unit tests
* [ ] Documentation ([N/A])
* [X] Filenames and locations

File: src/extensions/Array.test.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+describe('Array.prototype.toObject', async (should: any) => {
+  const { assert } = should()
+
+  {
+    const arr = [['key1', 'value1'], ['key2', 'value2']]
+    const cb = (arr: ReadonlyArray<any>) => ({
+      key: arr[0],
+      value: arr[1]
+    })
+    assert({
```
Comment (warrenv) [inline src/extensions/Array.test.ts]: Can we add an empty line here? It will keep the formatting consistent with the rest of the file.

**Review by kennylavender — DISMISSED** [review]
* [N/A] Tested changes manually
* [x] Checked accidental architectural/style changes
* [x] Reviewed entire diff
* [x] Unit tests
* [N/A] Documentation
* [x] Filenames and locations

**Comment by jwicks31** [comment]
@kennylavender @lautarodragan sorry I dismissed your reviews with the last commit. It's just an extra empty line.

**Review by kennylavender — APPROVED** [review]

**Review by wzalazar — APPROVED** [review]

**Pull request merged**